keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -492,6 +483,32 @@ These are the common steps to delete a `Bucket`. Note: there are atypical workfl
 + If the release policy is "Retain" then the `Bucket` remains "Released" and it can potentially be reused.
 + When the COSI controller sees the `Bucket`'s `conditions[x].Reason is "Deleted", it deletes all the finalizers and the real deletions occur.
 
+###  Setting Access Permissions
+#### Dynamic Provisioning
+Incoming BucketAccessRequest either contains a ServiceAccountName where cloud provider supports identity integration or a accessSecretName. In both cases, the incoming BAR representing a request for a user to access the `Bucket`.
+When requesting access to a bucket, workloads will go through the following  scenarios described here:
++  New User and New/Existing Bucket: In this scenario, we do not have user account in the backend storage system as well as no access for this user to the `Bucket`. 
+	+ Create user account in the backend storage system 
+	+ add the access privileges for the user to the `Bucket`
+	+ return the credentials to the workload owning the BAR
++  Existing User and New Bucket: In this scenario, we have the user account already created and exists in the backend storage system but the account is not associated to the `Bucket`. 
```

Comment:
"...In this scenario, we have the user account already created and exists in the backend storage system but the account is not associated to the `Bucket`. "  -->

"In this scenario, the user account already exists in the backend storage system, but the account is not associated to the `Bucket`. "

Reply:
Done